src/benchmarks/micro/libraries/System.Collections/Dictionary/Perf_Dictionary.cs

```diff
@@ -25,22 +25,5 @@ public void InitializeContainsValue()
 
         [Benchmark]
         public object Clone() => new Dictionary<int, int>(_dict);
```

Comment:
The original recommendation was to remove this method and make a Clone benchmark for all collections.

There is already a CtorFromCollection benchmark which largely covers this case and includes immutable collections.

What that benchmark doesn't test is the optimization immutable collections can have where `CreateRange` on another instance of the same type of immutable collection can just return the input. However, that doesn't seem worth benchmarking as it can easily be unit-tested.